datanode/gateway/graphql/schema.graphql

```diff
@@ -368,6 +366,24 @@ type LiquidityProviderFeeShare {
   virtualStake: String!
 }
 
+"The SLA statistics for each liquidity provider"
+type LiquidityProviderSLA {
```

Comment:
This should be in the MarketData and MarketDataObservable types as well I supposem like the LiquidityProviderFeeShare are.